system/logaccess/Azure/LogAnalytics/CurlClient/AzureLogAnalyticsCurlClient.cpp

```diff
@@ -166,32 +183,52 @@ static void submitKQLQuery(std::string & readBuffer, const char * token, const c
         CURLcode                curlResponseCode;
         OwnedPtrCustomFree<curl_slist, curl_slist_free_all> headers = nullptr;
         char                    curlErrBuffer[CURL_ERROR_SIZE];
-        curlErrBuffer[0] = '\0';
 
         char * encodedKQL = curl_easy_escape(curlHandle, kql, strlen(kql));
```

Comment:
I think the null termination warning was here - and was a false positive.

Reply:
Agreed, the scan might not have access to the CURL_ERROR_SIZE definition.
honestly, I wasn't ready for code review, but glad you jumped on.